src/core/api/common/mapper-module.ts

```diff
@@ -0,0 +1,43 @@
+import { CityWeatherInfo } from 'src/app/shared/interfaces/city-weather-info.interfaces';
```

Comment:
We have two options create utils class for this or create service.
For example rename mapper-module to weather-transform.service